### DISCUSSION TITLE
What is the "account_sql" column for?

### DISCUSSION THREAD

**Discussion by greenreader9** [NXTS-Developers/MOFHY-Lite#36] [Q&A] [answered]
What is the use of the "account_sql" column in the "hosting_account" table?
Thanks

**Comment by mahtab2003** [comment, marked as answer]
This is sql column is for sql hostname

**Reply by greenreader9** [reply]
So where is it used then? On viewaccount.php, it still only displayed sqlxxx.mydomain.com

**Reply by mahtab2003** [reply]
If you are using callback then it will work or else it will be the same

**Reply by greenreader9** [reply]
Got it
